src/workerd/io/trace.h

```diff
@@ -397,6 +480,31 @@ class Exception final {
   kj::Maybe<kj::String> stack;
 
   void copyTo(rpc::Trace::Exception::Builder builder);
+  Exception clone();
+};
+
+struct Resume final {
```

Comment:
Can you add a documentation/comment to all newly added struct, explaining their purpose and what they represent?